packages/snage/src/note/reporter.ts

```diff
@@ -0,0 +1,40 @@
+import {getFunctionName, ValidationError} from 'io-ts';
+
+export enum ReportMode {
+    Simple,
+    Complex,
```

Comment:
Vielleicht
```suggestion
    WithPath,
```
oder so ähnlich?